infrastructure/modules/container-app-environment/tfdocs.md

```diff
@@ -38,6 +32,14 @@ Type: `string`
 
 The following input variables are optional (have default values):
 
+### <a name="input_private_dns_zone_rg_name"></a> [private\_dns\_zone\_rg\_name](#input\_private\_dns\_zone\_rg\_name)
+
+Description: Name of the hub resource group where the private DNS zone is located. This is only required if adding custom DNS records.
```

Comment:
Maybe something like:
This is only required when hosting container apps with an HTTP ingress